src/System.ServiceModel.Federation/src/System/ServiceModel/Federation/WsTrustTokenParameters.cs

```diff
@@ -24,20 +25,25 @@ public class WSTrustTokenParameters : IssuedSecurityTokenParameters
         private TimeSpan _maxIssuedTokenCachingTime = DefaultMaxIssuedTokenCachingTime;
         private MessageSecurityVersion _messageSecurityVersion;
         private int _issuedTokenRenewalThresholdPercentage = DefaultIssuedTokenRenewalThresholdPercentage;
-        private string _target;
 
         /// <summary>
-        /// Values that are used to obtain a token from an IdentityProvider
+        /// Instantiates aa <see cref="wsTrustTokenParameters"/> that describe the parameters for a WSTrust request.
         /// </summary>
```

Comment:
typo: aa

Reply:
fixed in: b160df7f33caf6447f14e47b81ab3ff9f5930924